pkg/sql/pgwire/testdata/pgtest/timezone

```diff
@@ -52,8 +52,8 @@ Query {"String": "SET TIME ZONE \"America/Chicago\""}
 until
 ReadyForQuery
 ----
-{"Type":"ParameterStatus","Name":"TimeZone","Value":"America/Chicago"}
 {"Type":"CommandComplete","CommandTag":"SET"}
+{"Type":"ParameterStatus","Name":"TimeZone","Value":"America/Chicago"}
```

Comment:
i feel like some other test files need to change as a result of this change. how did they pass `pgwire` datadriven tests before o_o

Reply:
this file wasn't passing against PG before 🤥 

let's see which others turn up